src/types.ts

```diff
@@ -26,13 +27,12 @@ export type Entity = {
   metadata?: EntityMetadata
 }
 
-export type ServerVersion = string
 export type ServerName = string
 export type ServerAddress = string
 
 export type ServerStatus = {
   name: ServerName
-  version: ServerVersion
+  version: EntityVersion
```

Comment:
This would be something like `default version`?

Reply:
I don't understand the question. `EntityVersion` is an enum on this same file

Reply:
I wasn't sure what this field was used for, but now I get it!